src/common/veteran.js

```diff
@@ -642,4 +643,68 @@ const completeVeteran = {
   campLejeune: true
 };
 
-module.exports = { blankVeteran, completeVeteran };
+function veteranToApplication(veteran) {
+  // TODO(awong): Figure out how to do this w/o going through JSON.
+  return JSON.stringify(veteran, (key, value) => {
+    switch (key) {
+      // Convert radio buttons into booleans.
+      case 'isVaServiceConnected':
+      case 'compensableVaServiceConnected':
+      case 'provideSupportLastYear':
+      case 'receivesVaPension':
+      case 'provideFinancialInfo':
+      case 'understandsFinancialDisclosure':
+      case 'sameAddress':
+      case 'cohabitedLastYear':
+      case 'isCoveredByHealthInsurance':
+      case 'isMedicaidEligible':
+      case 'isEnrolledMedicarePartA':
+        return value.value === 'Y';
```

Comment:
by default are the values N if not submitted or will these just become undefined?